librdmacm/rsocket.c

```diff
@@ -118,6 +118,10 @@ static struct rs_svc listen_svc = {
 	.run = listen_svc_run
 };
 
+static uint32_t pollcnt;
+static bool suspendpoll;
+static int pollsignal[2];
```

Comment:
0 is a valid socket, shouldn't this be inited to -1?

Reply:
I was relying on that both fd's can't be 0.  But I can update these to -1.